app/src/main/java/tech/pauly/findapet/data/models/AnimalListResponse.java

```diff
@@ -14,7 +14,7 @@
     @Element(name = "header")
     private Header header;
 
-    @Element
+    @Element(required = false)
```

Comment:
no animal response came back with no lastOffset too, so threw an element missing exception. I could just handle that exception, but it's cleaner to handle the animal response as is.